enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -0,0 +1,327 @@
+            volumeMounts:
+            - name: trusted-ca
+              mountPath: /etc/pki/ca-trust/extracted/pem
+              readOnly: true
+        - name: trusted-ca
+          configMap:
+            name: trusted-ca
+            items:
+              - key: ca-bundle.crt
+                path: tls-ca-bundle.pem
+  ...
+  ```
+  * Create a file watcher that watches changes to the mounted trust bundle that kills the main operator process and
+    allows the backing k8s Deployment to start a new Pod that mounts the updated trust bundle. 
+    Implementation example: [cluster-ingress-operator](https://github.com/openshift/cluster-ingress-operator/pull/334)
+  * But then the same concern about avoiding unnecessary reconciliation remains, as in the [previous section](#configuring-custom-trusted-certificates): 
+    how to detect if the trust bundle file changed or the operator just restarted for another reason?
+---
+
+### Test Plan & Infrastructure Needed
+
+In addition to unit testing individual WMCO packages and controllers, an e2e will be added to the WMCO repo for the
+master/release-4.14 branches. We will leverage the [existing proxy workflow](https://steps.ci.openshift.org/workflow/openshift-e2e-aws-proxy)
+from the release repo and add a job to the WMCO repo's CI config to test with. We will run regression testing to make
+sure the addition of the egress proxy feature does not break existing functionality. If we release a community offering
+with this feature, we will need to add a CI job using a cluster-wide proxy on OKD.
+
+### Open Questions
+
+> 1. Is testing cluster-wide proxy in CI on __only AWS__ okay? This is consistent with what the Linux side does.
+     QE may want to cover all platforms though.
+> 2. Can we use the existing [CI proxy job](https://github.com/openshift/release/tree/master/ci-operator/step-registry/ipi/conf/aws/proxy)
+     to test an HTTPS proxy that requires custom certs to access, or can we only test HTTP and no-proxy using it? If the
+     latter, we'll need to explore how to do this in EC2 and add improvements to the linked workflow.
+> 3. Do user-created Windows workloads need proxy config set in their spec? Does this already happen for Linux
+     counterparts? e.g. setting proxy env vars and mounting the trustedCA ConfigMap as a volume to the deployment spec
+     Regardless, I don't think this should be WMCO's responsibility, so is this on another team or upon the user?
+  <!-- TODO: update non-goals based on the answer to number 3  -->
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+A community version of WMCO 8 or 9 can be released with incremental additions to Windows proxy support
+functionality, giving users an opportunity to get an early preview of the feature using OKD/OCP 4.13 or 4.14.
+It will also allow us to collect feedback to troubleshoot common pain points and learn if there are any shortcomings.
+
+#### Tech Preview -> GA
```

Comment:
I would remove this sub-heading and just having `Graduation criteria` where you capture all of this.